spec/std/http/spec_helper.cr

```diff
@@ -0,0 +1,46 @@
+require "spec"
+
+private def wait_for(timeout = 5.seconds)
+  now = Time.monotonic
+
+  until yield
+    Fiber.yield
+
+    if (Time.monotonic - now) > timeout
+      raise "server failed to start within 5 seconds"
```

Comment:
I'm just seeing this now, for better re-usability this should report the actual `timeout` value.

Reply:
@straight-shoota 👍 I'll fix that up when i get home.

Reply:
I suppose this will be ready to be merged first and #7197 needs to be rebased anyway.